### PR TITLE
EVG-6448: small UI changes

### PR DESCRIPTION
**Pull request by dominoweir**
This set of changes includes updates to the styling of various components, the removal of unused/nonexistent CSS classes, and a fix to the patch loading order issue.

### CODE REVIEW

File: src/components/build/LogContainer.tsx [hunk 1]
```diff
@@ -105,6 +113,11 @@ export class LogContainer extends React.Component<Props, State> {
       }
     }, this.props.task.task_id, logType, this.props.task.execution);
   }
+
+  private lineContainsError = (logLine: string) => {
```
Comment (john-m-liu) [inline src/components/build/LogContainer.tsx]: Sorry I hadn't thought of this earlier, but there's a different API that tells us what the severity of each line is (https://evergreen.mongodb.com/json/task_log/spruce_ubuntu1604_lint_37ec5816d14e05bd02535788d114da8b97ed0231_19_07_11_18_09_18/0?type=T). I don't think it's worth prioritizing fixing at the moment, but could you add a TODO comment here about changing it?

File: src/components/login/Login.tsx [hunk 2]
```diff
@@ -83,6 +88,9 @@ export class Login extends React.Component<Props, State> {
             </Button>
           </DialogActions>
         </Dialog>
+        <Snackbar open={this.state.snackbarOpen} message={this.state.snackbarMessage}
+          anchorOrigin={{ vertical: "bottom", horizontal: "left", }} onClose={this.onCloseSnackbar}
+          action={[<IconButton key="close" color="inherit" onClick={this.onCloseSnackbar}><CloseIcon /></IconButton>,]} />
```
Comment (john-m-liu) [inline src/components/login/Login.tsx]: Accidental comma at the end?

File: src/components/patch/Patch.tsx [hunk 3]
```diff
@@ -33,7 +34,8 @@ export class Patch extends React.Component<Props, State> {
       datetime: datetime,
       project: this.props.Patch.Version.identifier,
       author: this.props.Patch.Version.author,
-      builds: this.props.Patch.Builds
+      builds: this.props.Patch.Builds,
+      reconfigureLink: "https://evergreen.mongodb.com/patch/" + this.props.Patch.Version.id
```
Comment (john-m-liu) [inline src/components/patch/Patch.tsx]: Get the root of the URL from the ui_url part of the config for this

File: src/components/build/BuildView.tsx [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
 import { APITask, Build, ConvertToAPITasks, ConvertToBuild } from 'evergreen.js/lib/models';
 import * as React from 'react';
 import * as rest from "../../rest/interface";
 import '../../styles.css';
 import BuildSidebar from './BuildSidebar';
 import LogContainer from './LogContainer';
 
+const StyledDrawer = withStyles({
+  paper: {
+    top: 65,
```
[the remark this answers is not in the document]
Comment (dominoweir) [reply src/components/build/BuildView.tsx]: I thought so too, but apparently not. When I add in px, my editor does not recognize the unit and produces an error instead.

File: src/components/build/LogContainer.tsx (hunk 1 above)
Comment (dominoweir) [reply src/components/build/LogContainer.tsx]: Done

File: src/components/patch/Patch.tsx (hunk 3 above)
Comment (dominoweir) [reply src/components/patch/Patch.tsx]: Fixed

File: src/components/app/App.tsx [hunk 5]
```diff
@@ -39,48 +61,51 @@ export class Evergreen extends React.Component<Props, State> {
...
-      </div>
+      <ThemeProvider theme={theme}>
+        <div className="app">
+          <UserContextConsumer>
+            {() => {
+              return (
+                <HashRouter>
+                  <AppBar className="app-bar" >
+                    <Toolbar>
+                      <img src={"https://evergreen.mongodb.com/static/img/evergreen_green.png"} className="app-icon" />
```
[the remark this answers is not in the document]
Comment (dominoweir) [reply src/components/app/App.tsx]: Fixed

File: src/components/login/Login.tsx (hunk 2 above)
Comment (dominoweir) [reply src/components/login/Login.tsx]: Fixed

File: src/components/build/BuildView.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (john-m-liu) [reply src/components/build/BuildView.tsx]: It works in mine and things compile. Are you making sure to make it a string?

[the remark this answers is not in the document]
Comment (dominoweir) [reply src/components/build/BuildView.tsx]: Oh, I didn't know it was supposed to be a string. Fixed now.

**Pull request merged**